Game/src/Collision.java

```diff
@@ -63,31 +63,87 @@ public StaticRect isCollidingTop(Player player){
     }
 
     /**
-     * Bottom edge of player is colliding.
+     * Player Collision Bottom
      * @param player The player.
      * @return The bottom edge of the player is colliding with any static map object (StaticRect) in sceneObjects.
      */
-    public StaticRect isCollidingBottom(Player player){
+    public StaticRect isCollidingBottom(Player player) {
 
-        for( Object object : MainApplication.sceneObjects){
-            if( object instanceof  StaticRect) {
+        for (Object object : MainApplication.sceneObjects) {
+            if (object instanceof StaticRect) {
                 StaticRect staticRect = (StaticRect) object;
 
-                if (staticRect.checkCollision(player).contains(StaticRect.CollisionType.Bottom)) {
+                if (staticRect.checkCollision(player).contains(StaticRect.CollisionType.Bottom) && staticRect.getCollisionEnabled()) {
                     return staticRect;
                 }
             }
         }
         return null;
     }
 
-    public boolean enemyCollision(Player player){
-        for(Enemies enemy : MainApplication.enemies){
-            if(enemy.checkCollision(player)){
+    /**
+     *
+     * @param staticRect Any StaticRect.
+     * @param player The player.
+     * @return Player is colliding with staticRect.
+     */
+    public boolean isColliding(StaticRect staticRect, Player player){
+        return (staticRect.checkCollision(player).contains(StaticRect.CollisionType.Right) ||
+                staticRect.checkCollision(player).contains(StaticRect.CollisionType.Left) ||
+                staticRect.checkCollision(player).contains(StaticRect.CollisionType.Top) ||
+                staticRect.checkCollision(player).contains(StaticRect.CollisionType.Bottom));
```

Comment:
I'll leave this one for you to think about. What is the optimal ordering of these statements?